### PR TITLE
Improve media player device class description

### PR DESCRIPTION
**Pull request by theberko**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/media_player.markdown [hunk 1]
```diff
@@ -183,5 +183,5 @@ Allows to group media players together for synchronous playback. Only works on s
 The way media players are displayed in the frontend can be modified in the [customize section](/getting-started/customizing-devices/). The following device classes are supported for media players:
 
 - `tv`: Device is a television type device.
-- `speaker`: Device is speaker or stereo type device.
-- `receiver`: Device is audio video receiver type device taking audio and outputting to speakers and video to some display.
+- `speaker`: Device is a speaker or stereo type device.
+- `receiver`: Device is an audio video receiver type device taking audio and outputting to speakers and video to some displays.
```
Comment (MartinHjelmare) [inline source/_integrations/media_player.markdown]:
```suggestion
- `receiver`: Device is an audio/video receiver type device taking audio and outputting to speakers and video to displays.
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Comment by MartinHjelmare** [comment]
My comment with a change suggestion hasn't been addressed.

**Comment by theberko** [comment]
Oops. It's my first time editing a page and I'm not really sure how this works. Your version is definitely better than mine.

**Comment by MartinHjelmare** [comment]
You should be able to commit my suggestion directly here in the GitHub web user interface by clicking the commit suggestion button under my suggestion above.

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**